.travis.yml

```diff
@@ -9,4 +9,6 @@ install:
 
 script:
   - npm test
-  - ./node_modules/sw-testing-helpers/project/publish-docs.sh master
+  - if [[ "$TRAVIS_BRANCH" = "master" && "$TRAVIS_OS_NAME" = "linux" && "$TRAVIS_PULL_REQUEST" = "false" ]]; then
```

Comment:
MOre out of curiosity than anything: Is there a constellation where `$TRAVIS_BRANCH = "master"` and `$TRAVIS_PULL_REQUEST = "true"`?